parm/config/gcafs/yaml/defaults.yaml

```diff
@@ -0,0 +1,76 @@
+base:
+  DOIAU: "NO"
+  DO_JEDIATMVAR: "NO"
+  DO_JEDIATMENS: "NO"
+  DO_JEDIOCNVAR: "NO"
+  DO_JEDISNOWDA: "NO"
+  DO_MERGENSST: "NO"
+  DO_STARTMEM_FROM_JEDIICE: "NO"
+  DO_GOES: "NO"
+  DO_BUFRSND: "NO"
+  DO_GEMPAK: "NO"
+  DO_AWIPS: "NO"
+  DO_NPOESS: "NO"
+  DO_TRACKER: "NO"
+  DO_GENESIS: "NO"
+  DO_GENESIS_FSU: "NO"
+  DO_METP: "YES"
```

Comment:
Should this be `NO`?

Reply:
Probably, @bbakernoaa any comments?

Reply:
yes METP should be no by default but will likely want to be able to include it for development purposes as @CoryMartin-NOAA mentioned previously